core/src/test/java/com/linecorp/armeria/server/VirtualHostBuilderTest.java

```diff
@@ -327,4 +328,16 @@ void precedenceOfDuplicateRoute() {
         final Route route = serviceConfig.route();
         assertThat(route).isSameAs(routeA);
     }
+
+    @Test
+    void multipartUploadsLocationCustomization() {
+        final VirtualHost h1 = new VirtualHostBuilder(Server.builder(), false)
+                .multipartUploadsLocation(FileSystems.getDefault().getPath("logs", "access.log"))
+                .build(template, noopDependencyInjector);
+        assertThat(h1.multipartUploadsLocation().getFileName().toString()).isEqualTo("access.log");
```

Comment:
Could extract the path you passed with `multipartUploadsLocation()` above into a local variable and call `isEquals` with it.